internal/version/version.go

```diff
@@ -130,26 +127,9 @@ func (v Info) Proto() (*pb.NodeVersion, error) {
 	}, nil
 }
 
-// containsVersion compares the allowed version array against the passed version
-func containsVersion(all []SemVer, x SemVer) bool {
-	for _, n := range all {
-		if x == n {
-			return true
-		}
-	}
-	return false
-}
-
-// StrToSemVerList converts a list of versions to a list of SemVer
-func StrToSemVerList(serviceVersions []string) (versions []SemVer, err error) {
-	for _, subversion := range serviceVersions {
-		sVer, err := NewSemVer(subversion)
-		if err != nil {
-			return nil, err
-		}
-		versions = append(versions, *sVer)
-	}
-	return versions, err
+// matchesVersion compares and checks if the passed version is greater/equal than the minimum required version
+func matchesVersion(test SemVer, target SemVer) bool {
+	return test.Major > target.Major || (test.Major == target.Major && test.Minor > target.Minor) || (test.Minor == target.Minor && test.Patch >= target.Patch)
```

Comment:
```
test.Major > target.Major || 
(test.Major == target.Major && test.Minor > target.Minor) || 
(test.Minor == target.Minor && test.Patch >= target.Patch)
```

Storage node is running v0.13.5 and our minimum version is v1.13.0. -> false || false || true. The last check is not checking the major version.